packages/react-app/src/lib/bridge.js

```diff
@@ -141,12 +141,12 @@ export const relayTokens = async (ethersProvider, token, amount) => {
 };
 
 export const transferTokens = async (ethersProvider, token, amount) => {
-  const confirmsPromise = fetchConfirmations(token.chainId);
+  const confirmsPromise = fetchConfirmations(token.chainId, ethersProvider);
   const txPromise = isxDaiChain(token.chainId)
     ? transferAndCallToken(ethersProvider, token, amount)
     : relayTokens(ethersProvider, token, amount);
   const totalConfirms = parseInt(await confirmsPromise, 10);
   const tx = await txPromise;
 
-  return [tx, totalConfirms];
+  return [tx, totalConfirms, ethersProvider];
```

Comment:
why are we returning ethersProvider here ?

Reply:
ah good catch, not sure why that was there, thanks